guide/deployment/configuration.mdx

```diff
@@ -49,6 +49,7 @@ The following table shows the environment variables required by each service. Be
 | `BF_URL`             | The `botfront-api` root url                                                                                                                                                                              | `rasa` `actions`          |
 | `API_KEY`            | GraphQL API key. You can then set the `authorization` header to the `API_KEY` value to perform GraphQL operations                                                                                        | `botfront`                |
 | `MODELS_LOCAL_PATH`  | Where the trained model returned by Rasa is stored locally. Defaults to `/app/models/` and should not be changed in a containerized environment. The Botfront Dockerfile exposes a volume with that path | `botfront` (optional)     |
+| `GRAPHQL_REQUEST_SIZE_LIMIT` | The size over which requests sent to the GraphQL server will receive a HTTP Error 413 (Payload Too Large) response. [Defaults to 100kb.](https://github.com/expressjs/body-parser#limit) | `botfront` (optional)
```

Comment:
Didn't you say you increased the default to 200kb?

Reply:
Oops.